package-lock.json

```diff
@@ -45,6 +45,8 @@
         "moment-timezone": "~0.5.31",
         "morgan": "~1.9.0",
         "multer": "~1.3.1",
+        "ngraph.graph": "~20.0.0",
```

Comment:
All the changes to other packages should be okay but need to verify that is correct by trying code. It is curious that the ngraph package was added as it already was in package.json but I think it might have been missed before.